packages/sources/css/src/components/snackbar/src/index.css

```diff
@@ -0,0 +1,31 @@
+@import '@vtmn/css-design-tokens/src/legacy-colors';
+@import '@vtmn/css-design-tokens/src/spacings';
+@import '@vtmn/css-design-tokens/src/typography';
+@import '@vtmn/css-design-tokens/src/opacities';
+@import '@vtmn/css-design-tokens/src/shadows';
+
+.vtmn-snackbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: rem(8px) rem(16px);
+  height: rem(48px);
+  max-width: fit-content;
+  font-family: var(--vtmn-typo_font-family);
+  font-size: var(--vtmn-typo_text-2-font-size);
+  line-height: var(--vtmn-typo_text-2-line-height);
+  color: var(--vtmn-semantic-color_content-primary-reversed);
+  background-color: var(--vtmn-semantic-color_background-primary-reversed);
+  box-shadow: var(--vtmn-shadow_200);
+  border-radius: rem(4px);
```

Comment:
border radius is dependent of the root font size ? should it be constant and defined in px instead ?

Reply:
Thanks for feedback Xavier. I created an issue to rework the pixels sizes.